cmd/daily-lifecycle-ops.go

```diff
@@ -150,8 +151,29 @@ func lifecycleRound(ctx context.Context, objAPI ObjectLayer) error {
 					// Do nothing, for now.
 				}
 			}
+
 			// Deletes a list of objects.
-			objAPI.DeleteObjects(ctx, bucket.Name, objects)
+			deleteErrs, err := objAPI.DeleteObjects(ctx, bucket.Name, objects)
+			if err != nil {
+				logger.LogIf(ctx, err)
+			}
+
+			for i := range deleteErrs {
+				if deleteErrs[i] != nil {
+					logger.LogIf(ctx, deleteErrs[i])
+					continue
+				}
+				// Notify object deleted event.
+				sendEvent(eventArgs{
+					EventName:  event.ObjectRemovedDelete,
+					BucketName: bucket.Name,
+					Object: ObjectInfo{
+						Name: objects[i],
+					},
+					Host: "[INTERNAL]",
```

Comment:
Can the host also have information like `Expiry ILM` or something like that? Currently, that is the only option, not sure if there are other options. The only other one that I can think of is `Post Object Restore`

Reply:
> Can the host also have information like `Expiry ILM` or something like that? Currently, that is the only option, not sure if there are other options. The only other one that I can think of is `Post Object Restore`

Something like `[ILM-EXPIRY]` ?

Reply:
yes we need to be able to differentiate

Reply:
Once that is added, we can take this in

Reply:
was @abperiasamy okay with adding this feature? @kannappanr

Reply:
Yes